Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8 AS builder
+FROM quay.io/fedora/fedora:34-x86_64 AS builder
 
```

Comment:
There are still a lot of openshift specifics in this file. I'm wondering if we can drop it entirely.